### DISCUSSION TITLE
Leistung am Netzübergabepunkt / automatische Phasenumschaltung go-e

### DISCUSSION THREAD

**Discussion by humi208** [camueller/SmartApplianceEnabler#575] [Installation und Betrieb]
Hallo, 

gibt es eine Möglichkeit die aktuelle Leistung, Bezug oder Überschuss/Einspeisung, mittels Home Manager und dem SAE zu erhalten und den Wert dann zyklisch zu verteilen, z.B an die Wallbox?
So könnte man mittels des PGrid Parameters des go-e Charger die automatische Phasenumschaltung dieser WB nutzen. 

https://github.com/goecharger/go-eCharger-API-v2/discussions/110#discussioncomment-4973967

Für eine Hilfe wäre ich dankbar, wie ich an den Wert komme um diesen dann zyklisch zu setzen.

**Comment by thmsck** [comment]
Hallo,

eventuell hilft dir dies schon weiter?

https://github.com/datenschuft/SMA-EM

Viele Grüße

**Comment by camueller** [comment]
Es gab Überlegungen, [im SAE Unterstützung für eine automatische Umschaltung der Phasenanzahl einzubauen](https://github.com/camueller/SmartApplianceEnabler/issues/281), aber bisher ist da nichts passiert.

**Comment by humi208** [comment]
Die go-e macht die Phasenumschaltung ja selbstständig. Man muss ihr nur die Leistung/Überschuss zyklisch mitteilen

**Comment by camueller** [comment]
Bei anderen Wallboxen ist das gar nicht oder anders gelöst und der SAE kann nicht jedes Feature von jeder Wallbox unterstützen. Angesichts der installierten PVLeistung macht für die meisten User die Phasenumschaltung keinen Sinn, speziell beim Überschussladen.

**Comment by humi208** [comment]
Ich hätte den der Parameter auch selbst eingetragen ;)
Ich hatte halt für Hoffnung, dass man im SAE bereits an solche Werte, wie die Grid Leistung als Variable rankommt. Dann wäre es ein leichtes gewesen diese zyklisch zu setzen/senden.

**Comment by camueller** [comment]
Solche Werte hat der SAE nicht, sondern nur das EMS (also der SHM).

**Comment by humi208** [comment]
Danke, ich denke mit dem ersten Post kann ich dann etwas aufbauen.

@thmsck 
Danke für den Tipp

**Comment by Trunks1982** [comment]
Hi im iobroker semp Adapter haben wir das so gelöst das wir einfach die Umschalt Adresse angeben, bei go e psm.. Wenn dann die Ladeleistung eine gewisse Zeit zb 4min über bei 11kw 3600w beträgt soll sie auf 3phasen umschalten.Eigentlich ganz easy umgesetzt und es funktioniert wunderbar.

**Comment by humi208** [comment]
Um zwischen einphasig und dreiphssig umzuschalten gehört schon etwas mehr und das ist nicht so easy. Manche Autos vertragen das umschalten nicht und etwas Logik für eine funktionierende Hysterese braucht man auch. All das macht die go-e von sich aus.
Sie benötigt eben nur den aktuellen Überschuss am Übergabepunkt. Den hat der EMS und muss einfach nur an die WB gesendet werden...

**Comment by Trunks1982** [comment]
Ich weiss das der Go E das selber macht.Wie gesagt ich habs ja mit dem Entwickler beim Semp Adapter umgesetzt.Und brauchten nur die Ladeleistung und nicht die Gridwerte.Der SHM gibt vor mit wieviel geladen werden soll un dann kann man das einbauen das er wie gesagt bei 3600w Ladeleistung nach x min jmschalten soll.

und es geht nur über den http befehl psm.Und nicht anders.

**Comment by humi208** [comment]
Ok, so könnte es auch gehen, nach Vorgabe des SHM. Dann musst du psm setzen und den Ladestrom abhängig von der Vorgabe des SHM
